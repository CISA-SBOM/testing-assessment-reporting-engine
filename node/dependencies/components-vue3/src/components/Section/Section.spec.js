import { describe, expect, it } from 'vitest'
import { shallowMount } from '@vue/test-utils'
import Component from './Section.vue'

describe('Section', () => {
  it('should match its default snapshot', () => {
    const wrapper = shallowMount(Component, {})
    expect(wrapper.element).toMatchSnapshot()
  })
})
