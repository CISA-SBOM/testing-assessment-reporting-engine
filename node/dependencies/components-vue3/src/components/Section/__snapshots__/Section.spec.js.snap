// Vitest Snapshot v1

exports[`Section > should match its default snapshot 1`] = `
<div
  class="block bg-white dark:bg-gray-800 dark:border-gray-700"
  data-id="sds-section"
>
  <header
    class=""
  >
    <div
      class="flex px-4 py-3"
    >
      <div
        class="self-center flex-grow"
      >
        <!---->
        <!---->
      </div>
      <!---->
    </div>
  </header>
  <!---->
</div>
`;
